README.md

```diff
@@ -9,17 +9,18 @@ Unchained is a multi-blockchain backend interface with three main goals:
 
 ## Table Of Contents
 
-- [Project Details](#project-details)
+- [Introduction](#introduction)
+- [Table Of Contents](#table-of-contents)
 - [Helpful Docs](#helpful-docs)
 - [Coin Stack Components](#coin-stack-components)
+- [Architecture Diagrams](#architecture-diagrams)
 - [Notes](#notes)
 - [Local Networking](#local-networking)
+- [Setup](#setup)
 - [Docker-Compose Local Dev Instructions](#docker-compose-local-dev-instructions)
-  - [Prerequisites](#prerequisites)
-  - [Running](#running)
-- [Kubernetes Local Dev Instructions](#kubernetes-local-dev-instructions)
-  - [Docker Desktop (macOS)](docs/docker-desktop.md)
-  - [Minikube (Linux)](docs/minikube.md)
+    - [Prerequisites](#prerequisites)
+    - [Running](#running)
+- [Common Issues](#common-issues)
```

Comment:
To my surprise, VSCode markdown plugin generates this based on the actual content on file save

Reply:
very cool!